message/message.go

```diff
@@ -218,3 +218,21 @@ func (nc *NetworkCurve) UnmarshalJSON(b []byte) error {
 
 	return nil
 }
+
+const PreAuthEndpoint = "/v1/enduser-auth/preauth"
```

Comment:
suuuuuper nit: i would call it Preauth instead of PreAuth just because there is no hyphen or space in the actual endpoint name.